pkg/chartutil/values.go

```diff
@@ -64,16 +74,6 @@ func (v Values) Table(name string) (Values, error) {
 	return table, err
 }
 
-// AsMap is a utility function for converting Values to a map[string]interface{}.
-//
-// It protects against nil map panics.
-func (v Values) AsMap() map[string]interface{} {
```

Comment:
Functions cannot be removed, as third parties may rely on this functionality. Instead, a note that this function is deprecated and will be removed in Helm 4 is preferred.